src/irc/core/irc-commands.c

```diff
@@ -1045,6 +1045,11 @@ void irc_commands_init(void)
 	command_bind_irc("knockout", NULL, (SIGNAL_FUNC) cmd_knockout);
 	command_bind_irc("setname", NULL, (SIGNAL_FUNC) command_1self);
 	command_bind_irc("server purge", NULL, (SIGNAL_FUNC) cmd_server_purge);
+	/* common shorthands for MSG NickServ, ChanServ, OperServ, MemoServ :parameter */
+	command_bind_irc("ns", NULL, (SIGNAL_FUNC) command_self);
+	command_bind_irc("cs", NULL, (SIGNAL_FUNC) command_self);
+	command_bind_irc("os", NULL, (SIGNAL_FUNC) command_self);
+	command_bind_irc("ms", NULL, (SIGNAL_FUNC) command_self);
```

Comment:
You need to unbind the commands too, but I'm wondering if it wouldn't be better to add those as default aliases?

Reply:
Place to add the aliases: https://github.com/irssi/irssi/blob/master/irssi.conf
But lets let the maindevs have the final say

Reply:
Added unbind calls for now, until someone tells me how to add default aliases. :)